service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -158,10 +157,6 @@ public void createAiNotebookInstanceDo() throws Exception {
         controlledResourceService.getControlledResource(
             workspace.getWorkspaceId(), resource.getResourceId(), user.getAuthenticatedRequest()));
 
-    // TODO(PF-643): this should happen inside WSM.
```

Comment:
Unlike the other waits, this test makes calls to GCP using the original workspace owner. I think the waiting is not necessary in that case, but if we see flakiness in this connected test we should revisit that.